docs/pulses.ipynb

```diff
@@ -53,7 +53,11 @@
    "outputs": [],
    "source": [
     "pulse = tof.Pulse(\n",
-    "    neutrons=1_000_000, tmin=1.0 * ms, tmax=3.0 * ms, lmin=1.0 * AA, lmax=10.0 * AA\n",
+    "    neutrons=1_000_000,\n",
+    "    t_start=1.0 * ms,\n",
+    "    t_end=3.0 * ms,\n",
+    "    wav_min=1.0 * AA,\n",
+    "    wav_max=10.0 * AA,\n",
```

Comment:
Mixing start/end and min/max seems like something no one will ever be able to remember. Can we use one convention for both `t` and `wav`?

Reply:
Yeah, I think I will go with min/max :+1: